### PR TITLE
create new superadmin

### PR DESCRIPTION
**Pull request by inaod568**
Adding config to have a superadmin email added too.

- [ ] Tests passing
- [ ] Black formatting
- [ ] Migrations for any changes to the database schema
- [ ] Rebase/merge the `dev` branch
- [ ] Note in the CHANGELOG

### CODE REVIEW

File: dds_web/web/user.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ def confirm_2fa():
     # Valid 2fa initiated token, but user does not exist (should never happen)
```
Comment (alneberg) [inline dds_web/web/user.py:204]: I guess you could change this message if you expect it to actually happen.

**Review by alneberg — APPROVED** [review]
Looks good to me.

File: migrations/versions/666003748d14_change_active_nullable_and_default.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    session = Session(bind=op.get_bind())
+    all_user_rows = session.query(models.User).all()
+    for user in all_user_rows:
+        if user.active is None:
+            user.active = False
+    session.commit()
+    op.alter_column("users", "active", existing_type=mysql.TINYINT(display_width=1), nullable=False)
```
Comment (talavis) [inline migrations/versions/666003748d14_change_active_nullable_and_default.py:30]: Not sure: should the new default value be mentioned here, or is that only in the model?

**Review by talavis — APPROVED** [review]
It seems to work fine

File: migrations/versions/666003748d14_change_active_nullable_and_default.py (hunk 2 above)
Comment (inaod568) [reply migrations/versions/666003748d14_change_active_nullable_and_default.py:30]: It seems like it. Not adding / looking into this now though

**Pull request merged**